### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by hyl246**

### CODE REVIEW

**Comment by ph691** [comment]
@hyl246 Can we remove the redundant protocols.txt now as part of this PR please?

File: README.md [hunk 1]
```diff
@@ -1,9 +1,84 @@
...
+
+* `NEWACCOUNT <new account name> <account type (savings/current)> <opening balance>` - Create new account
+
+  e.g. `NEWACCOUNT Home savings 10000` create a 'Home' savings account with $10,000
+
+* `MOVE <Amount> <From> <To>` - Transfer money from user's interal accounts
+
+  e.g. `MOVE 5000 Main Home` transfer $5,000 from account Main to account Home
+
+* `SUBTRACTMONEYFROMACCOUNT <From> <Amount>`
```
Comment (ph691) [inline README.md:34]: Missing ADDMONEYTOACCOUNT <To> <Amount>

**Comment by hyl246** [comment]
> @hyl246 Can we remove the redundant protocols.txt now as part of this PR please?

I think we can remove the txt file

File: README.md (hunk 1 above)
Comment (hyl246) [reply README.md]: added ADDMONEYTOACCOUNT

**Review by ph691 — APPROVED** [review]

**Pull request merged**